src/components/cellteststatuswidget.cpp

```diff
@@ -0,0 +1,134 @@
+#include "cellteststatuswidget.h"
+
+CellTestStatusWidget::CellTestStatusWidget(channelInfo info, QFrame *parent) : QFrame(parent)
+{
+
+    statusBarLayout = new QHBoxLayout;
+    statusBarLayout->setSpacing(0);
+
+    // Create the Warmup Label Text
+    char warmupLabelText[100];
```

Comment:
Is there a reason you used c style strings here and not QString (and there totally might be)? If not really, I would suggest changing this all to use QStrings. I personally find them easier, they support Unicode, apparently better performance, etc. And would likely fix the build break.

Reply:
I tried using QStrings initially, but I was getting an error that it couldn't convert to a c string, but since revisiting this, I realized it was just the tr() function that needed a c string, so we should be good now.